README.md

```diff
@@ -206,6 +206,26 @@ to switch to the vty console frontend optimized for screen readers, run
     cabal run -fvty LambdaHack
 
 
+Building and running under Windows with stack
+---------------------------------------------
+I managed to build and run it by using the stack build rather than the cabal build. I installed sdl2, sdl-ttf, and sdl-mixer using the technique here (https://www.reddit.com/r/haskellgamedev/comments/4jpthu/windows_sdl2_is_now_almost_painless_via_stack/) _for each library_, not just for sdl2.
```

Comment:
Is that "sdl2-ttf"? Is sdl-mixed really needed? Do you mean, you installed the haskell libraries? I'm pretty sure this should not be necessary, because they are built as needed when you build LambdaHack. However, installing the C SDL libraries is required, so you were right to do it. I can be done with one commands as in

https://github.com/LambdaHack/LambdaHack/blob/master/appveyor.yml#L44

or the same with stack.